### PR TITLE
Event notifications and email template.

### PR DESCRIPTION
**Pull request by materiywakal**

### CODE REVIEW

File: src/ReKreator/ReKreator.BL/NotificationHandler.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                    {
+                        continue;
+                    }
+
+                    if (favoriteEvent.NotificationPeriodsBeforeEvent.HasFlag(flag))
+                    {
+                        if ((favoriteEvent.EventHolding.Time - DateTime.Now).TotalHours
+                            <= _dictionary.Container[flag])
+                        {
+                            await SendMessageAboutFavorite(favoriteEvent.User, favoriteEvent.EventHolding.Event.Title,
```
Comment (alexfromny) [inline src/ReKreator/ReKreator.BL/NotificationHandler.cs]: Async

File: src/ReKreator/ReKreator.BL/NotificationHandler.cs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+                _dbHandler.UserRepository.Update(user);
+                await _dbHandler.SaveAsync();
+            }
+        }
+
+        private async Task SendMessageAboutFavorite(User to, string title, DateTime time)
+        {
+            await _sender.MessageToUserAsync(to, "Favorite Event",
+                $"<div>Event {title} will start {time.ToString("dd.MM.yyyy")} at {time.TimeOfDay}.</div>");
```
Comment (alexfromny) [inline src/ReKreator/ReKreator.BL/NotificationHandler.cs]: we have date format

Comment (materiywakal) [reply src/ReKreator/ReKreator.BL/NotificationHandler.cs]: those formats placed in UI layer, while i using mine in BLL

Comment (materiywakal) [reply src/ReKreator/ReKreator.BL/NotificationHandler.cs]: Should i move all dateformats to BLL?

**Pull request merged**